pdksync.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_development_dependency 'bundler', '~> 1.15'
+  spec.add_development_dependency 'bundler', '~> 2.0.1'
```

Comment:
Can we keep the bunder version the same just make sure everything works.
Any we can upgrade bundler with a new PR. Other than that everything looks good.Thank you.